contracts/credit/CreditFacadeV3.sol

```diff
@@ -288,32 +288,25 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
     /// @param calls List of calls to perform before liquidating the account
     /// @dev When the credit facade is paused, reverts if caller is not an approved emergency liquidator
     /// @dev Reverts if `creditAccount` is not opened in connected credit manager
-    /// @dev Reverts if account is not liquidatable
+    /// @dev Reverts if account has no debt or is neither unhealthy nor expired
     /// @dev Reverts if remaining token balances increase during the multicall
     function liquidateCreditAccount(address creditAccount, address to, MultiCall[] calldata calls)
         external
         override
         whenNotPausedOrEmergency // U:[FA-2,12]
         nonReentrant // U:[FA-4]
     {
-        address borrower = _getBorrowerOrRevert(creditAccount); // U:[FA-5]
```

Comment:
`borrower` is removed from `LiquidateCreditAccount` event, and the check is now performed in `calcDebtAndCollateral`